test-failure-policy

```diff
@@ -109,6 +117,22 @@ def checkRetry(trace):
 # -----------------------------------------------------------------------------
 # Known Issue Matching and Filing
 
+
+def check_all_known_issues(api, trace, image):
+    img = None
+    number = None
+    naughty_dir = os.path.join(BOTS_DIR, "naughty")
+
+    for img in os.listdir(naughty_dir):
```

Comment:
This will also iterate over symlinks, and half of the entries are links. Please move to https://docs.python.org/3/library/os.html#os.scandir and filter out links. This happens a lot in tests and check_known_issue() is expensive, so not being overly inefficient here matters IMHO.

Reply:
Good point, having fast CI is worth the effort.